kcc_cli/keyboard_backlight.py

```diff
@@ -0,0 +1,89 @@
+import subprocess
+from time import sleep
+from typing import Dict
+
+from kcc_cli.common import read_file, write_file
+from kcc_cli.enums import Position, Mode
+
+
+def get_laptop_model() -> str:
+    return subprocess.check_output(['sudo', 'dmidecode', '-s', 'system-product-name']).decode('utf-8').strip()
```

Comment:
It may be better to use this `sudo dmidecode -t 1 | grep Version:` instead so that we can set the paths per model like the `oryp4`.